common/src/main/scala/explore/schemas/odb/implicits.scala

```diff
@@ -264,6 +267,60 @@ object implicits {
       }
   }
 
+  implicit class NonNegDurationOps(val nnd: NonNegDuration) {
```

Comment:
`extends AnyVal` ?

Reply:
Unfortunately, the compiler does not allow that. `value class may not wrap another user-defined value class`